spec/import_spec.rb

```diff
@@ -153,20 +167,24 @@ def upload_file!(name, ext='csv')
           expect(page).to have_content "Successfully imported 1 author"
           expect(Author.count).to eq(1)
         end
+
       end
 
 
       context "without headers" do
         context "with known csv headers" do
-          let(:options) {
-            {template_object: ActiveAdminImport::Model.new(csv_headers: ['Name', 'Last name', 'Birthday'])}
-          }
+
+          let(:options) do
+            attributes = {csv_headers: ['Name', 'Last name', 'Birthday']}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.